frontend/src/components/GameInvite.vue

```diff
@@ -52,28 +52,35 @@ const receivedInviteList = ref();
 const router = useRouter();
 
 setTimeout(() => {
-  socket.emit("getReceivedGameInvites", storeUser.state.user.id);
+  socket.emit("getReceivedGameInvites", { data: storeUser.state.user.id });
 }, 100);
 
-socket.on("getReceivedGameInvites", (response) => {
-  receivedInviteList.value = response;
-});
+onMounted(() => {
```

Comment:
Sometimes the game is not rendered.
The steps to reproduce:
1. Send the game invite to another user.
2. Join the queue.
3. The other user ignores your invitation and clicks 'Find a Match'

==>> the game starts but it doesn't render(sometimes it does for one player)

Reply:
I've checked the previous version, it was stable

Reply:
As discussed, this is due to the un-finished game flow, this will be updated in future PRs.